docs/users.md

```diff
@@ -341,27 +341,29 @@ the URL for the server as `https://kubeflex-control-plane:<nodeport>`
 
 
 ## Control Plane Types
-
-At this time KubFlex supports the following control plane types:
-
-- k8s: this is the stock Kube API server with a subset of controllers running in the controller manager.
-- ocm: this is the [Open Cluster Management Multicluster Control Plane](https://github.com/open-cluster-management-io/multicluster-controlplane), which provides a basic set of capabilities such as
-clusters registration and support for the [`ManifestWork` API](https://open-cluster-management.io/concepts/manifestwork/).
-- vcluster: this is based on the [vcluster project](https://www.vcluster.com) and provides the ability to create pods in the hosting namespace of the hosting cluster.
-- host: this control plane type exposes the underlying hosting cluster with the same control plane abstraction
-used by the other control plane types.
-- external: this control plane type represents an existing cluster that was not created by KubeFlex and is not the KubeFlex hosting cluster.
+At this time KubeFlex focuses documentation on the following supported control plane types:
```

Comment:
I think the existing wording ("At this time KubFlex supports the following control plane types") is just fine. We are removing support for the `ocm` type.